dcc-submission-ui/dictionary/app/scripts/helpers/dictionary-utils.js

```diff
@@ -1,4 +1,4 @@
-/* globals $ */
+/* globals $,jsondiffpatch */
```

Comment:
Or this:

https://github.com/hortonworks/polestar/blob/36d19a47f4a0bdf402c215da74e9371159de4aab/src/app/index.js#L26